class/defaults.yml

```diff
@@ -20,11 +83,32 @@ parameters:
       password: '?{vaultkv:${customer:name}/${cluster:name}/${_instance}/admin-password}'
     # Replica count
     replicas: 2
+    # TLS configuration
+    tls:
+      variant: certmanager
+      termination: passthrough
```

Comment:
I did took the assumption, that as we are doing maintenance on clusters anyway. But it's an valid point and the description https://hub.docker.com/r/jboss/keycloak/ "Those files need to be mounted in /etc/x509/https directory. The image will automatically convert them into a Java keystore and reconfigure Wildfly to use it." doesn't sound like a timed reload does happen.

Reply:
Hm, pity. Problem is, Project Syn shouldn't be VSHN specific, i.e. we can't assume that other Syn users do maintenance often enough to restart Keycloak to reload LE certs "by coindicence". If `passthrough` is default, IMO we should definitely make users aware of this limitation in the documentation. Otherwise, maybe the `reencrypt` option with a long-running self-signed cert with CA validation might be the better option for generic usage. We could still override this to `passthrough` in the VSHN defaults.

Reply:
Sound like a problem for something like https://argoproj.github.io/argo-rollouts/features/restart/, https://github.com/stakater/Reloader, https://github.com/jetstack/cert-manager/issues/1168. Let's see if I can figure out more.